mpd/tests.py

```diff
@@ -844,6 +683,169 @@ def test_command_list(self):
             ('xfade', '5')], sorted(res[5].items()))
 
 
+# MPD client tests which do not mock the socket, but rather replace it
+# with a real socket from a socket
+class TestMPDClientSocket(unittest.TestCase):
```

Comment:
Thanks. This looks good! Can you make those tests being skipped on `windows`?

Reply:
Done, asterisk: I skip these tests specifically whenever `socketpair` is not available. From my brief testing, that means Python 2 on Windows. They pass with my Python 3.6 install on Windows.